tgui/packages/tgui/format.js

```diff
@@ -140,3 +140,14 @@ export const formatTime = (time, msg = "") => {
 export const truncate = (str, n = 25) => {
   return (str.length > n) ? str.substr(0, n-1) + '…' : str;
 };
+
+/**
+ * Converts numbers to a shorter format to save space
+ */
+const Suffixes = ["", "k", "M", "B", "T"];
```

Comment:
what are these suffixes? thousand, million, etc?

Reply:
Yeah they could be used for money

Reply:
why not just use `formatSiUnit`

Reply:
I didn't want to show any decimal numbers